packages/manager/modules/product-offers/src/workflows/product-offers-services-workflow.class.js

```diff
@@ -32,24 +32,27 @@ export default class ServicesWorkflow extends Workflow {
   }
 
   /**
-   * Get the pricings from available detach plan codes,
+   * Get the pricings from available detach or upgrade plan codes,
    * based on the pricing type
    */
   getPricings() {
-    const [detachPlancode] = this.detachPlancodes; // Yes, for the moment, there is just one plan code
-    this.plancode = detachPlancode.planCode;
-    this.pricings = this.computePricing(detachPlancode.prices);
+    const [actionPlancode] = this.plancodes; // Yes, for the moment, there is just one plan code
+    this.plancode = actionPlancode.planCode;
+    this.pricings = this.computePricing(actionPlancode.prices);
 
     if (this.hasUniquePricing()) {
       this.$timeout(() => {
         this.currentIndex += 1;
         [this.pricing] = this.pricings;
+        if (typeof this.onPricingSubmit === 'function') {
```

Comment:
Is this verification necessary? Based on the name "onPricingSubmit" it sounds like it'll always be a function unless it's null or undefined, no?

Reply:
It is to test also if `onSubmitFunction` is a function and not an object. But you're right its name clearly indicates it must be a function. It's mainly to replace `this.onPricingSubmit && isFunction(onPricingSubmit)` kind of test we usually used to do with Lodash.